plugins/topology/src/common/components/ResourceName.tsx

```diff
@@ -46,7 +46,7 @@ export const ResourceIcon = ({ className, kind }: ResourceIconProps) => {
 
 export type ResourceNameProps = {
   kind: string;
-  name: string;
+  name: React.ReactNode;
```

Comment:
```suggestion
  name: React.ReactNode | string;
```

In some places string has been passed to this prop

Reply:
@debsmita1 `ReactNode` can be of type `string` as well.

Reply:
```
    type ReactText = string | number;
    type ReactChild = ReactElement | ReactText;
    type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;

```